isic_archive/api/dataset.py

```diff
@@ -337,7 +338,7 @@ def initiateZipUploadToS3(self, dataset, params):
             raise ValidationException('Signature must be specified.', 'signature')
 
         try:
-            return Dataset().initiateZipUploadS3(dataset=dataset, signature=signature, user=user)
+            return Dataset().initiateZipUploadS3(dataset=dataset, signature=signature, user=user, filename=params.get('filename', None))
```

Comment:
Could you extract `filename` as a separate variable? It may make tracing errors from Sentry easier.

Reply:
Fixed. Though FWIW this wouldn't help with Sentry since params is in the local scope.